README.md

```diff
@@ -62,9 +62,19 @@ Alternatively, if you do not wish to change anything in the Dockerfile or the pr
 docker build -t cbioportal-image https://github.com/thehyve/cbioportal-docker.git
 ```
 
-If you want to change any variable defined in portal.properties, have a look [here](adjusting_portal.properties_configuration.md). If you want to build an image based on a different branch, you can read [this](adjusting_Dockerfile_configuration.md). 
```

Comment:
Thanks for pointing it out. It was a mistake when solving the merging conflict. I will introduce this part again!